cloud_foundry_api/datadog_checks/cloud_foundry_api/constants.py

```diff
@@ -4,7 +4,7 @@
 import semver
 
 SOURCE_TYPE_NAME = "Cloud Foundry"
-MAX_LOOKBACK_SECONDS = 600
+DEFAULT_MAX_LOOKBACK_SECONDS = 60
```

Comment:
What do you think of keeping the default `600` as to remain backwards compatible?

Reply:
The default `600` is one of the main causes of this issue. The window is simply too large. When a check run fails, this means that we only need to look back at the last 15 seconds (the default collection interval of the agent checks). There is no reason to go back 10 minutes into the past. A window of 60 seconds would cover the same failure cases but with a smaller collision window.